EIPS/eip-4626.md

```diff
@@ -87,17 +87,13 @@ MUST return the address of a token implementing the ERC-20 standard.
 #### calculateShares
 `function calculateShares(uint256 underlyingAmount) public view returns (uint256 shareAmount)`
 
-Returns the amount of vault shares corresponding to a given underlying amount.
-
-`calculateShares(calculateUnderlying(shareAmount))` MUST equal `shareAmount`
+Returns the amount of vault shares corresponding to a given underlying amount, for the `withdraw` method. The return value is only guaranteed to be exact if executed immediately before a `withdraw` call in the same transaction.
     
 #### calculateUnderlying
    
 `function calculateUnderlying(uint256 shareAmount) public view returns (uint256 underlyingAmount)`;
 
-Returns the amount of underlying tokens corresponding to a given amount of vault shares.
-
-`calculateUnderlying(calculateShares(underlyingAmount))` MUST equal `underlyingAmount`
+Returns the amount of underlying tokens corresponding to a given amount of vault shares, for the `mint` method. The return value is only guaranteed to be exact if executed immediately before a `mint` call in the same transaction.
```

Comment:
not following, why `mint` specifically? and wouldnt it be immediately **after**?